_partials/integrations/hibernate-ogm.html.haml

```diff
@@ -0,0 +1,3 @@
+The Hibernate team maintains the
+%a{:href=> "http://hibernate.org/ogm"}Hibernate OGM
```

Comment:
Please use http://hibernate.org/ogm/ with a slash at the end. Let's avoid a redirection.